nautobot_ssot/tests/test_contrib.py

```diff
@@ -309,6 +309,68 @@ class Adapter(NautobotAdapter):
             "Custom fields aren't properly loaded through 'BaseAdapter'.",
         )
 
+    def test_default_get_queryset(self):
+        """Test default 'get_queryset' method."""
+
+        class TenantModel(NautobotModel):
+            """Test model for testing default 'get_queryset' method."""
+
+            _model = Tenant
+            _modelname = "tenant"
+            _identifiers = ("name",)
+            _attributes = ("description",)
+
+            name: str
+            description: str
+
+        class Adapter(NautobotAdapter):
+            """Test adapter for testing default 'get_queryset' method."""
+
+            top_level = ("tenant",)
+            tenant = TenantModel
+
+        new_tenant_name = "NASA"
+        Tenant.objects.create(name=new_tenant_name)
+        adapter = Adapter()
+        adapter.load()
+        diffsync_tenant_1 = adapter.get(TenantModel, new_tenant_name)
+        diffsync_tenant_2 = adapter.get(TenantModel, "Test")
+
+        self.assertEqual(new_tenant_name, diffsync_tenant_1.name)
+        self.assertEqual("Test", diffsync_tenant_2.name)
```

Comment:
Can we drop this test? I am not sure it provides extra coverage in comparison to the rest of the existing tests, as they will be implicitly testing the default implementation. Thoughts?

Reply:
Yeah I was thinking the same thing as I wrote it. I think it helped me figure out the structure of the test so I ended up leaving it. I'll yank it out. Thanks!